### PR TITLE
Logistics multiple fixes

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

### CODE REVIEW

**Review by karenkrieger — CHANGES_REQUESTED** [review]
Arrumar os erros do Spectral

**Comment by Isabella-Veloso** [comment]
> Arrumar os erros do Spectral

Feito.

**Comment by Isabella-Veloso** [comment]
> Arrumar os erros do Spectral

Feito na maior parte.

File: VTEX - Logistics API.json [hunk 1]
```diff
@@ -3255,14 +3152,20 @@
...
-                                                        "cost": {
-                                                            "type": "number"
-                                                        }
+                                                            "type": "string",
+                                                            "description": "Code that identifies a dock.",
+                                                            "default": "dock1"
+                                                                  },
+                                                        "time":   {
+                                                            "type": "string",
+                                                            "description": "This field returns the dock's processing time in days and hours. If none is configured, the call returns `00:00:00`.",
```
Comment (karenkrieger) [inline VTEX - Logistics API.json:3161]:
```suggestion
                                                            "description": "This field returns the dock's processing time in days and hours. If none is configured, the request returns `00:00:00`.",
```

File: VTEX - Logistics API.json [hunk 2]
```diff
@@ -3271,7 +3174,22 @@
                                                     }
                                                 },
                                                 "description": ""
```
[the remark this answers is not in the document]
Comment (Isabella-Veloso) [reply VTEX - Logistics API.json]: Feito.

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**